packages/datetime/src/common/classes.ts

```diff
@@ -17,6 +17,10 @@ export const DATEPICKER_FOOTER = "pt-datepicker-footer";
 export const DATEPICKER_MONTH_SELECT = "pt-datepicker-month-select";
 export const DATEPICKER_YEAR_SELECT = "pt-datepicker-year-select";
 
+export const DATERANGEINPUT = "pt-daterangeinput";
+export const DATERANGEINPUT_FIELD = "pt-daterangeinput-field";
```

Comment:
// Copied from the other PR (sorry for the split contexts)

`pt-daterangeinput-input` is more correct, but it sounds a little silly. Happy to change to that if we think it's better though.